staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/sample_and_watermark.go

```diff
@@ -57,6 +57,7 @@ func (spg SampleAndWaterMarkPairVec) NewForLabelValuesSafe(initialWaitingDenomin
 	}
 }
 
+//nolint:golint,unused
```

Comment:
If it's unused, let's remove it...

Reply:
Yes, that's coming later in a cleanup PR.

Reply:
done